include/quda_define.h.in

```diff
@@ -29,6 +29,123 @@
 #undef QUDA_HETEROGENEOUS_ATOMIC
 #endif
 
+#cmakedefine QUDA_DIRAC_NDEG_TWISTED_CLOVER
```

Comment:
As discussed previously I am not a big fan of a monolithic define file for recompile/ ccache efficiency. Happy to have that for apps that use QUDA but we should aim for something more fine grained. But for another PR.

Reply:
Yep, understood.  This change doesn't change the needle in any regard, it moves the prior compiler flags to a preconfigured flag.

I made this change to fix bugs in test codes: the enablement flags were not passed to the tests, which meant that some functions would evaluate differently internally and externally of QUDA.

Reply:
Makes sense. We need to put some work in to really improve here.